src/services/requestExport.js

```diff
@@ -0,0 +1,59 @@
+import { logger } from '@dojot/dojot-module-logger';
+import requestDevice from './requestsDevice';
```

Comment:
I particularly don't think these file names are meaningful. When you use it, like `requestDevice.requestDevice`, it is not clear what is happening, _maybe_ something like `DeviceManager.get`, `FlowManager.set` become more meaningful.